libraries/AP_HAL_F4BY/AP_HAL_F4BY_Namespace.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace F4BY {
+	class F4BYScheduler;
+	class F4BYUARTDriver;
+	class F4BYStorage;
+	class F4BYRCInput;
+	class F4BYRCOutput;
+	class F4BYAnalogIn;
+	class F4BYAnalogSource;
+	class F4BYUtil;
+        class F4BYGPIO;
+        class F4BYDigitalSource;
+        class NSHShellStream;
+        class F4BYI2CDriver;
+        class F4BY_I2C;
+        class Semaphore;
+}
```

Comment:
Thank you, Francisco. We have a fix for namespaces already so I just checked once more we need to do that.

That's for "_full new HAL very similar to PX4_" I'm not quite sure I can see your point. Would you be so kind to clarify it? We implemented it as a separate HAL due to hardware difference in F4BY. For example, we have no i/o processor and there other differences also. In case a full new HAL doesn't seems as reasonable approach for this case what do you see as a more reasonable way?